libdbus-sys/build_vendored.rs

```diff
@@ -0,0 +1,389 @@
+//! Vendored Libdbus build.
+//! 
+//! Configures + Compiles + Statically Links a pinned libdbus release.
+use std::collections::HashMap;
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::process::Command;
+use std::path::Path;
+use std::error::Error;
+use std::string::ToString;
+use std::collections::hash_map::Iter;
+
+// Version of vendored libdbus
+const DBUS_VERSION: &str = "\"1.14.4\"";
+const DBUS_VERSION_MAJOR: u32 = 1;
+const DBUS_VERSION_MINOR: u32 = 14;
+const DBUS_VERSION_MICRO: u32 = 4;
+
+// Location of dbus-daemon socket
+const DBUS_SYSTEM_BUS_DEFAULT_ADDRESS: &str = "/run/dbus/system_bus_socket";
```

Comment:
These three constants should also have a reference

Reply:
Documented in 84b3ddf35a391205ce7340325c5b01879ced1359